<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>vendorWorkBench</class>
 <widget class="QWidget" name="vendorWorkBench">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vendor Workbench</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="_nameLit">
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="_name">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="_termsLit">
         <property name="text">
          <string>Default Terms:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>_terms</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="XComboBox" name="_terms">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="allowNull">
          <bool>true</bool>
         </property>
         <property name="type">
          <enum>XComboBox::Terms</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="_typeLit">
         <property name="text">
          <string>Type:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>_vendType</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="XComboBox" name="_vendType">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="allowNull">
          <bool>true</bool>
         </property>
         <property name="type">
          <enum>XComboBox::VendorTypes</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="XComboBox" name="_shipvia">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="allowNull">
          <bool>true</bool>
         </property>
         <property name="type">
          <enum>XComboBox::ShipVias</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="_shipviaLit">
         <property name="text">
          <string>Default Ship Via:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>_shipvia</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="VendorCluster" name="_vend">
           <property name="label">
            <string>Vendor #:</string>
           </property>
           <property name="nameVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_crmacct">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Account...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="_active">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Active</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="_close">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_print">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Print</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_edit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="_summaryGroup">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="_firstPurchaseLit">
        <property name="text">
         <string>First Purchase:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>_firstPurchase</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="DLineEdit" name="_firstPurchase">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="_lastYearsPurchasesLit">
        <property name="text">
         <string>Last Year's Purchases:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="XLabel" name="_lastYearsPurchases">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="_lastPurchaseLit">
        <property name="text">
         <string>Most Recent Purchase:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>_lastPurchase</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="DLineEdit" name="_lastPurchase">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="_ytdPurchasesLit">
        <property name="text">
         <string>YTD Purchases:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="XLabel" name="_ytdPurchases">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="_backlogLit">
        <property name="text">
         <string>Backlog:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="XLabel" name="_backlog">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="_openBalanceLit">
        <property name="text">
         <string>Open Balance:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="XLabel" name="_openBalance">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="_tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="_contactTab">
      <attribute name="title">
       <string>Contacts</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>12</number>
         </property>
         <item>
          <widget class="QRadioButton" name="_contact1Button">
           <property name="text">
            <string>Contact 1</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="_contact2Button">
           <property name="text">
            <string>Contact 2</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QStackedWidget" name="_stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="_contact1Page">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="ContactCluster" name="_primaryContact">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="accountVisible" stdset="0">
              <bool>false</bool>
             </property>
             <property name="activeVisible" stdset="0">
              <bool>false</bool>
             </property>
             <property name="initialsVisible" stdset="0">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>662</width>
               <height>103</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="_contact2Page">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="ContactCluster" name="_secondaryContact">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="accountVisible" stdset="0">
              <bool>false</bool>
             </property>
             <property name="activeVisible" stdset="0">
              <bool>false</bool>
             </property>
             <property name="initialsVisible" stdset="0">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>639</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_poTab">
      <attribute name="title">
       <string>Purchase Orders</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6"/>
     </widget>
     <widget class="QWidget" name="_receiptsTab">
      <attribute name="title">
       <string>Receipts and Returns</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7"/>
     </widget>
     <widget class="QWidget" name="_payablesTab">
      <attribute name="title">
       <string>Payables</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8"/>
     </widget>
     <widget class="QWidget" name="_cmTab">
      <attribute name="title">
       <string>Credit Memos</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9"/>
     </widget>
     <widget class="QWidget" name="_historyTab">
      <attribute name="title">
       <string>A/P History</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5"/>
     </widget>
     <widget class="QWidget" name="_checksTab">
      <attribute name="title">
       <string>Checks</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContactCluster</class>
   <extends>QWidget</extends>
   <header>contactcluster.h</header>
  </customwidget>
  <customwidget>
   <class>DLineEdit</class>
   <extends>QWidget</extends>
   <header>datecluster.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VendorCluster</class>
   <extends>QWidget</extends>
   <header>vendorcluster.h</header>
  </customwidget>
  <customwidget>
   <class>XComboBox</class>
   <extends>QComboBox</extends>
   <header>xcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>XLabel</class>
   <extends>QLabel</extends>
   <header>xlabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_vend</tabstop>
  <tabstop>_crmacct</tabstop>
  <tabstop>_active</tabstop>
  <tabstop>_terms</tabstop>
  <tabstop>_vendType</tabstop>
  <tabstop>_shipvia</tabstop>
  <tabstop>_firstPurchase</tabstop>
  <tabstop>_lastPurchase</tabstop>
  <tabstop>_tabWidget</tabstop>
  <tabstop>_contact1Button</tabstop>
  <tabstop>_contact2Button</tabstop>
  <tabstop>_primaryContact</tabstop>
  <tabstop>_close</tabstop>
  <tabstop>_print</tabstop>
  <tabstop>_edit</tabstop>
  <tabstop>_secondaryContact</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_close</sender>
   <signal>clicked()</signal>
   <receiver>vendorWorkBench</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>809</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_vend</sender>
   <signal>valid(bool)</signal>
   <receiver>_crmacct</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_vend</sender>
   <signal>valid(bool)</signal>
   <receiver>_print</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_vend</sender>
   <signal>valid(bool)</signal>
   <receiver>_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>768</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
